thrift/replicator.thrift

```diff
@@ -222,3 +224,10 @@ struct MergeDLQMessagesRequest{
 struct MergeDLQMessagesResponse{
   10: optional binary nextPageToken
 }
+
+struct FailoverMarkerAttributes{
```

Comment:
Probably move to L111 so that it's grouped with other replication task types

Reply:
done